pkg/kubelet/prober/prober.go

```diff
@@ -110,19 +109,19 @@ func (pb *prober) probe(probeType probeType, pod *v1.Pod, status v1.PodStatus, c
 		if err != nil {
 			klog.V(1).Infof("%s probe for %q errored: %v", probeType, ctrName, err)
 			if hasRef {
-				pb.recorder.Eventf(ref, v1.EventTypeWarning, events.ContainerUnhealthy, "%s probe errored: %v", probeType, err)
+				pb.recorder.Eventf(ref, nil, v1.EventTypeWarning, "Unhealthy", "%s probe errored: %v", "probe", err)
```

Comment:
@wojtek-t - per the [event-style guide](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/event-style-guide.md) maybe we can add `pod` as a related to object (one use case would be to query all the events related to a specific pod)